plugins/woocommerce/client/blocks/assets/js/data/cart/index.ts

```diff
@@ -58,10 +58,16 @@ register( store );
 // Likewise, if we have a valid persistent cart, we can skip the request.
 // The only reliable way to check if the cart is empty is to check the cookies.
 window.addEventListener( 'load', () => {
+	const hasCachedCart = persistenceLayer.get();
```

Comment:
Why do you call it `hasCachedCart`? I think `cachedCart` would be more appropriate for what's happening.

Reply:
fixed in 76d869262cf96409dbb4a631b3ebbe8f84a1a668